chainregistry.go

```diff
@@ -751,3 +760,34 @@ func initNeutrinoBackend(chainDir string) (*neutrino.ChainService, func(), error
 
 	return neutrinoCS, cleanUp, nil
 }
+
+// parseHeaderStateAssertion parses the user-specified neutrino header state
+// into a headerfs.FilterHeader.
+func parseHeaderStateAssertion(state string) (*headerfs.FilterHeader, error) {
+	if len(state) == 0 {
+		return nil, nil
+	}
+
+	split := strings.Split(state, ":")
+	if len(split) != 2 {
+		return nil, fmt.Errorf("header state assertion %v in " +
```

Comment:
builds showing a vet error:
```
chainregistry.go:773:15:error: Errorf format %v reads arg #1, but call has 0 args (vet)
```

Reply:
Also would be nice to have the error return the expected format.

Reply:
:woman_facepalming: thanks, fixed!